general/administration/installing.md

```diff
@@ -37,142 +33,89 @@ The Jellyfin project and its contributors offer a number of pre-built binary pac
+3. Create persistent storage for configuration and cache data:
 
-These folders will be used by Jellyfin to store data in. By default everything inside a Docker container gets removed once you delete the container. By making these folders you are able to delete the Jellyfin container while keeping your data. Amazing!
+    Either create two persistent volumes:
 
-#### Running the Container
+    ```sh
+    docker volume create jellyfin-config
+    docker volume create jellyfin-cache
+    ```
 
-1. Get the latest image.
+    Or create two directories on the host and use bind mounts:
 
     ```sh
-    docker pull jellyfin/jellyfin
+    mkdir /path/to/config
+    mkdir /path/to/cache
     ```
 
-2. Start the server in one of the following ways
-
-    **Docker**
+4. Create and run a container in one of the following ways:
 
-    Running a Container is very easy (as you might have noticed while running the Hello world! Container). However when running a service you'll need to add some options to your docker run Command. Below is the full command and then it will be explained what the different parts mean.
+    **Using Docker command line interface:**
 
     ```sh
     docker run -d \
-     --volume /path/to/config:/config \
-     --volume /path/to/cache:/cache \
+     --volume jellyfin-config:/config \
+     --volume jellyfin-cache:/cache \
      --volume /path/to/media:/media \
      --user 1000:1000 \
      --net=host \
      --restart=unless-stopped \
      jellyfin/jellyfin
     ```
 
-    - `-d`: This means that the Docker Container will run in the background. This can also be changed to -it this will make the container interactive allowing you to execute commands directly inside the container.
-
-    - `--volume` /path/to/config:/config: This option connects the "/path/to/config" folder on your Host to the "/config" inside the container. You do not need to have any config present in /path/to/config as during the run command of the container Jellyfin will generate all of the necessary files. Everything in front of the colon is the of the config folder you've created in the previous step.
-
-    - `--volume` /path/to/cache:/cache: This option connects the "/path/to/cache" folder on your Host to the "/cache" inside the container.
-
-    - `--volume` /path/to/media:/media: This option connects the "/path/to/media" folder on your Host to the "/media" inside the container. This option has to be added in order for Jellyfin to find your media files. It is also possible that you have your media files spread across different places. In that case you can keep adding more volumes by using:
+    Replace `jellyfin-config` and `jellyfin-cache` with `/path/to/config` and `/path/to/cache` respectively if using bind mounts.
 
-    - `--volume` /path/to/anothermedia:/media2 & --volume /path/to/differentmedia:/media3 etc. add as many as you like!
+    To mount your media library read-only append ':ro' to the media volume:
```

Comment:
Should we add a note about this preventing creating nfo's? There is a feature request for metadata to be stored in an alternative folder instead of dB or media location.

Reply:
Yes makes sense.

Reply:
we could move the RO warning here instead of the beginning of the docker section.

This one
[!Note] There is currently an issue with read-only mounts in Docker. If there are submounts within the main mount, the submounts are read-write capable.